pkg/kubelet/prober/worker.go

```diff
@@ -218,6 +221,11 @@ func (w *worker) doProbe() (keepGoing bool) {
 		return true
 	}
 
+	// Record the end of the initialization phase.
+	if result == results.Success && !w.hasInitialized {
+		w.hasInitialized = true
```

Comment:
if kubelet crashes and restarts, what happens?  Do we suddenly allow some (potentially large) number of failures again?

Reply:
Yes, based on the current implementation.